Basic contour label implementation

AnniekStok opened the pull request:
Implement option for displaying nodes that are not in lineage/group as contours, instead of hiding them completely. This behavior can be toggled from the Edit Menu (for now, probably should move to 'settings' eventually).
Maybe more as inspiration / intermediate state until we have figured out what we want exactly for the visualization (issues #217 , #283)

<img width="932" height="495" alt="image" src="https://github.com/user-attachments/assets/fd932f83-7cbc-41c5-9f76-9ce2a2cede32" />
Lineage mode: yellow is in the selected lineage (and is the selected node), therefore it is the only 'filled' label, and it gets a highlight.
<img width="940" height="595" alt="image" src="https://github.com/user-attachments/assets/b051bdf3-39b1-4938-9ed1-244b91a6732b" />
Group mode: green & purple are in group and are therefore filled, yellow is selected but not in group, so it gets a highlight (and visible point) but no fill.

cmalinmayor commented on src/motile_tracker/data_views/views/layers/track_labels.py:
```diff
@@ -295,40 +296,65 @@ def _refresh(self):
+                    return None
+
+                if visible == "all":
+                    return 0.6
```
Make a class-level constant and use that, so we can change if needed.

AnniekStok commented:
I added a new tab to expose the opacity and contour settings to the user, and added radio buttons to change the mode, for convenience. 

<img width="497" height="471" alt="image" src="https://github.com/user-attachments/assets/ae2d104d-a9da-471e-bcfb-ac296d142d0b" />

Some settings were conflicting in different modes, e.g. we used to have background opacity=0, but then in contour mode we need the background to be visible, so I changed the default to 0.3, so you can see something at the very least. 
Also, I disabled changing the background opacity when you are in 'All' mode, because there are no background labels in that case. 
The option to 'fill' in contour mode is only displayed when layer.contour > 0,  and only available for the foreground and highlighted labels, as it is very inefficient and counterintuitive to 'fill' all labels while in contour mode. 

Still not entirely sold on it, let's discuss what potential changes might help. 

![visualization_widget](https://github.com/user-attachments/assets/de77fdad-0b92-4965-8019-8cce62e5e3cd)

cmalinmayor reviewed (changes requested):
Bug as mentioned when node is deleted while in group. But otherwise I'm happy with the actual contour label changes! Also happy if you want to simplify the USERS options while keeping the backend API fully configurable.

cmalinmayor reviewed (approved):
I tested it more thoroughly, including with orthoviews and 3d view, and everything looks great! 🍅 🦖
